src/adapter.ts

```diff
@@ -137,7 +137,21 @@ export class SequelizeAdapter implements Adapter {
     }
 
     /**
-     * removePolicy removes a policy rule from the storage.
+     * addPolicies adds a policyList rules to the storage.
+     */
+    public async addPolicies(sec: string, ptype: string, rules: string[][]) {
+        const lines: CasbinRule[] = [];
+        for (const rule of rules) {
+            const line = this.savePolicyLine(ptype, rule);
+            lines.push(line);
+        }
+        await this.sequelize.transaction(async tx => {
+            await CasbinRule.bulkCreate(lines.map(l => l.get({ plain: true })), { transaction: tx });
```

Comment:
I wonder if using `await CasbinRule.bulkCreate(lines, { transaction: tx });`  is working for avoid call `map` to improve performance.